packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -182,4 +94,74 @@ class ConsentAnalytics extends StatelessWidget {
       OnboardingFlowNavigator.getNextPage(OnboardingPage.CONSENT_PAGE),
     );
   }
+
+  Widget _buildButton(
+    BuildContext context,
+    Color btnColor,
+    String label,
+    IconData icon,
+    bool isAccepted,
+  ) {
+    final Size size = MediaQuery.of(context).size;
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    const Color shadowColor = Color.fromARGB(144, 0, 0, 0);
+    return ElevatedButton.icon(
+      style: ElevatedButton.styleFrom(
+        fixedSize: Size(
+          size.width * 0.9,
+          size.height * 0.05,
+        ),
+        elevation: 2,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(30.0),
+        ),
+        primary: btnColor,
+        shadowColor: shadowColor,
+      ),
+      onPressed: () {
+        _analyticsLogic(
+          isAccepted,
+          userPreferences,
+          localDatabase,
+          context,
+        );
+      },
+      icon: Icon(
+        icon,
+        color: WHITE_COLOR,
+        size: size.height * 0.05,
+      ),
+      label: Text(
+        label,
+        style: TextStyle(
+          fontWeight: FontWeight.bold,
+          color: WHITE_COLOR,
+          fontSize: size.height * 0.025,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildTextHeader(BuildContext context, String title) {
+    return Center(
```

Comment:
Please add a textAlign to center (cf https://github.com/openfoodfacts/smooth-app/issues/1438#issuecomment-1086640892)

Reply:
Changes done.